src/parser_error.rs

```diff
@@ -319,7 +319,7 @@ impl ExpectedBase64Error {
 
     /// If `true`, the length of the base64 string was invalid.
     fn caused_by_invalid_length(&self) -> bool {
-        matches!(self.decode_error, base64::DecodeError::InvalidLength)
+        matches!(self.decode_error, base64::DecodeError::InvalidLength(_))
```

Comment:
Check if this is correct (the above was suggested by clippy and it compiles but I haven't checked deeper than that yet)

Reply:
I think this is fine. We just match on whatever the length is.